lib/utils/ast-utils.js

```diff
@@ -60,6 +60,23 @@ function findPluginsByName(j, node, pluginNamesArray) {
 	});
 }
 
+
+/**
+ * Find plugins property in config and
+ * remove if it has no elements.
+ * @param  {any} j - jscodeshift API
+ * @param  {Node} rootNode - node to start search from
+ */
```

Comment:
Missing jsdoc for return here

Reply:
`It lookouts for the plugins property and, the array is empty, it removes it from the AST`